packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -223,7 +225,7 @@ $border: var(--onyx-1px-in-rem) solid var(--onyx-color-component-border-neutral)
     th,
     td {
       position: relative;
-      padding: var(--onyx-density-xs) var(--onyx-density-md);
+      padding: var(--onyx-table-padding-vertical) var(--onyx-table-padding-inline);
 
```

Comment:
imho, a simpler solution would suffice:
- no need for CSS variables
- no need for `calc()`

```suggestion
      position: relative;
      height: 2.5rem;
      padding: 0 var(--onyx-density-md);
      align-content: center;
      vertical-align: middle;
```

Reply:
But this does not scale with densities or custom paddings :)
Thats why I used calc instead so the height adapts automatically for all densities

Reply:
Ah, good point! I think in this case we can just use `box-sizing: content-box`:
```css
      padding: var(--onyx-density-xs) var(--onyx-density-md);
      box-sizing: content-box;
      height: var(--onyx-spacing-lg);
      align-content: center;
```

Reply:
Nope that doesn't work (see http://localhost:6006/?path=/story/data-datagrid-features--selection) since `th` is not a regular box element.

I'd stick to the calc solution if thats also fine for you :)